src/test/java/seedu/address/commons/util/HelpWindowUtilTest.java

```diff
@@ -0,0 +1,66 @@
+package seedu.address.commons.util;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import java.awt.Desktop;
+import java.awt.HeadlessException;
+import java.io.IOException;
+import java.net.URI;
+
+import org.junit.jupiter.api.Test;
+
+class HelpWindowUtilTest {
+
+    private static String validLink = "https://github.com/AY2122S2-CS2103-F11-2";
+    private static String invalidLink = "invalid link";
```

Comment:
Just a very minor suggestion, maybe the constants here can use `private static final` and make use of all upper case with `_` underscore for the constant name? :)

Reply:
missed that out! will change it in the next commit!

Reply:
wait @tiewweijian, would you want to add it in now? if not, we can approve and merge. your call!

Reply:
ok nvm. i think next one as this PR is linked to #41. charmaine created one i think you about it. let's approve this and merge and since charmaine gave 👍🏻, guess we can!

Reply:
I think we can wait though? lets see what @tiewweijian says

Reply:
Mm I think I'll push a new commit over and then y'all can review, approve and merge ! So that its clearer that this PR is for issue 41 ! 
I'll close issue 41 once yall approve and merge in too !!

Reply:
@tiewweijian i'll tag the issue #41 to this PR. @charmainehly shall we approve and merge? you could do it! 😄